Sources/StreamChatUI/Components.swift

```diff
@@ -37,6 +37,11 @@ public struct _Components<ExtraData: ExtraDataTypes> {
 
     /// Button used for sending a message, or any type of content.
     public var sendButton: UIButton.Type = ChatSendButton.self
+
+    /// A view to check/uncheck an option.
+    public var checkmarkControl: ChatCheckboxControl.Type =
+        ChatCheckboxControl.self
+
```

Comment:
The type here can be just `UIControl`?

Reply:
Yes you are right 👍

Reply:
Actually, no @VojtaStavik, because the ChatCheckboxControl has a label that we need to access it, and that it does not exist on UIControl